database/migrations/2022_03_02_045807_create_letter_outs_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateLetterOutsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('letter_outs', function (Blueprint $table) {
+            $table->uuid('id')->primary();
+            $table->string('no_surat');
+            $table->date('tgl_surat');
```

Comment:
buat tanggal surat, kasih cast di modelnya kek gini

protected $casts = [
    'tgl_surat' => 'datetime:d/m/Y', 
];

date yang lain juga samain pake $cast model masing-masing

Reply:
Okee okee

Reply:
lupa gw, yang ini juga ditambahin format
```suggestion
            $table->date('tgl_surat')->format('Y-m-d');
```